painting/src/main/java/com/ataulm/artcollector/painting/ui/PaintingActivity.kt

```diff
@@ -27,12 +34,46 @@ class PaintingActivity : AppCompatActivity() {
         val paintingId = intent.data!!.pathSegments.last()
         injectDependencies(PaintingId(paintingId))
 
+        // TODO: check with app links that it still works and doesn't care about postponed transitions
+        postponeEnterTransition()
+
+        intent.loadImageIfAvailable()
```

Comment:
first, we load the image from the url we passed through the intent extras (if it's available, because this activity can also be opened from an app link!)